src/headers/tomcrypt_pk.h

```diff
@@ -37,6 +37,16 @@ typedef struct Oid {
 
 int pk_get_oid(int pk, oid_st *st);
 
+typedef struct {
+  void* p;
+  unsigned long len;
+  int radix;
+} ltc_pk_part;
+
+#define PK_PART_HEX(s)     &((ltc_pk_part){s, 0, 16})
+#define PK_PART_DEC(d)     &((ltc_pk_part){d, 0, 10})
```

Comment:
Instead of `PK_PART_DEC(d)` I would define `PK_PART_RADIX(d,r)` so that we have: `PK_PART_HEX`, `PK_PART_RADIX` and `PK_PART_BIN`